iina/MPVController.swift

```diff
@@ -1024,18 +1028,23 @@ class MPVController: NSObject {
   private func readEvents() {
     queue.async {
       while ((self.mpv) != nil) {
-        let event = mpv_wait_event(self.mpv, 0)
+        let event = mpv_wait_event(self.mpv, 0)!
```

Comment:
Is it safe to force unwarp here?

Reply:
The change here is that the code was treating the return as option in `readEvents`, but then passed it to `handleEvent` which declared the type as `UnsafePointer<mpv_event>!`. So `readEvents` went to the trouble of treating it as optional only for it to be forced when `handleEvent` was called. Thus the forcing just got moved to an earlier time. I don't think there is a change as far as risking a crash due to force unwrapping.

If this is ever `nil` then mpv is seriously broken as the doc for `mpv_wait_event` says:
> The return value is never NULL.